diff --git a/lib/pal/PAL.cpp b/lib/pal/PAL.cpp
index d31966ec..9b2beac8 100644
--- a/lib/pal/PAL.cpp
+++ b/lib/pal/PAL.cpp
@@ -399,7 +399,7 @@ namespace PAL_NS_BEGIN {

 #if defined(__GNUC__) && !defined(__clang__)
 #pragma GCC diagnostic push
-#pragma GCC diagnostic ignored "-Wformat-truncation"  // error: 'T' directive output may be truncated writing 1 byte into a region of size between 0 and 16 [-Werror=format-truncation=]
+// #pragma GCC diagnostic ignored "-Wformat-truncation"  // error: 'T' directive output may be truncated writing 1 byte into a region of size between 0 and 16 [-Werror=format-truncation=]
 #endif
         (void)snprintf(buf, sizeof(buf), "%04d-%02d-%02dT%02d:%02d:%02d.%03dZ",
                        1900 + tm.tm_year, 1 + tm.tm_mon, tm.tm_mday,
