diff --git a/CMakeLists.txt b/CMakeLists.txt
index b6549ac1..d694aef4 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -100,7 +100,7 @@ if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC")
 set(WARN_FLAGS "/W4 /WX")
 else()
 # No -pedantic -Wno-extra-semi -Wno-gnu-zero-variadic-macro-arguments
-set(WARN_FLAGS "-Wall -Werror -Wextra -Wno-unused-parameter")
+set(WARN_FLAGS "-Wall -Werror -Wextra -Wno-unused-parameter -Wno-type-limits")
 endif()

 if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
